README.md

```diff
@@ -44,49 +44,50 @@ This repository contains core libraries, test frameworks, engineering systems, p
+- **INSTALL**: [![Install GitHub MCP in VS Code](https://img.shields.io/badge/VS_Code-Install_GitHub_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=github&config=%7B%22type%22%3A%20%22http%22%2C%22url%22%3A%20%22https%3A%2F%2Fapi.githubcopilot.com%2Fmcp%2F%22%7D) [![Install GitHub MCP in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_GitHub_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=github&config=%7B%22type%22%3A%20%22http%22%2C%22url%22%3A%20%22https%3A%2F%2Fapi.githubcopilot.com%2Fmcp%2F%22%7D&quality=insiders) [![Install GitHub MCP in Visual Studio](https://img.shields.io/badge/Visual_Studio-Install_GitHub_MCP_Server-C16FDE?style=flat-square&logo=visualstudio&logoColor=white)](https://aka.ms/vs/mcp-install?%7B%22name%22%3A%22github%22%2C%22type%22%3A%22http%22%2C%22url%22%3A%22https%3A%2F%2Fapi.githubcopilot.com%2Fmcp%2F%22%7D)
 
 ### <img height="18" width="18" src="https://github.githubassets.com/assets/GitHub-Mark-ea2971cee799.png" alt="GitHub Logo" /> GitHub Awesome-Copilot
 - **REPOSITORY**: [github/awesome-copilot](https://github.com/github/awesome-copilot)
 - **DESCRIPTION**: Community-contributed instructions, prompts, and configurations to help you make the most of GitHub Copilot.
 - **CATEGORY**: `DEVELOPER TOOLS`
 - **TYPE**: `Local`
-- **INSTALL**: [![Install Awesome Copilot MCP in VS Code](https://img.shields.io/badge/VS_Code-Install_Awesome_Copilot_MCP_Server-0098FF?logo=visualstudiocode&logoColor=white)](https://aka.ms/awesome-copilot/mcp/vscode) [![Install Awesome Copilot MCP in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Awesome_Copilot_MCP_Server-24bfa5?logo=visualstudiocode&logoColor=white)](https://aka.ms/awesome-copilot/mcp/vscode-insiders)
+- **INSTALL**: [![Install Awesome Copilot MCP in VS Code](https://img.shields.io/badge/VS_Code-Install_Awesome_Copilot_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://aka.ms/awesome-copilot/mcp/vscode) [![Install Awesome Copilot MCP in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Awesome_Copilot_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://aka.ms/awesome-copilot/mcp/vscode-insiders) [![Install in Visual Studio](https://img.shields.io/badge/Visual_Studio-Install-C16FDE?style=flat-square&logo=visualstudio&logoColor=white)](https://aka.ms/awesome-copilot/mcp/vs)
```

Comment:
![image](https://github.com/user-attachments/assets/b281c2c5-b9cb-472e-90ca-aa3499a14c7b)

This is inconsistent from the rest of the new buttons.